apps/web/app/(basenames)/api/proofs/cb1/route.ts

```diff
@@ -33,34 +33,35 @@ import type { NextApiRequest, NextApiResponse } from 'next';
  *   "discountValidatorAddress": "0x502df754f25f492cad45ed85a4de0ee7540717e7"
  * }
  */
-async function handler(req: NextApiRequest, res: NextApiResponse) {
+async function handler(req: NextRequest) {
   if (req.method !== 'GET') {
-    return res.status(405).json({ error: 'method not allowed' });
+    return NextResponse.json({ error: 'method not allowed' }, { status: 405 });
   }
-  const { address, chain } = req.query;
-  const validationErr = proofValidation(address, chain);
+  const address = req.nextUrl.searchParams.get('address');
+  const chain = req.nextUrl.searchParams.get('chain');
+  const validationErr = proofValidation(address ?? '', chain ?? '');
   if (validationErr) {
-    return res.status(validationErr.status).json({ error: validationErr.error });
+    return NextResponse.json({ error: validationErr.error }, { status: validationErr.status });
   }
   if (!trustedSignerPKey) {
-    return res.status(500).json({ error: 'currently unable to sign' });
+    return NextResponse.json({ error: 'currently unable to sign' }, { status: 500 });
```

Comment:
Just a thought, I know this stuff is here already but if you can't assume that constants like `trustedSignerPKey` are available, that's kinda scary. Failing the deploy if that stuff's not set seems like a better move. Def not something you need to address, though.

Reply:
good point. i'll look into this separately